brokerapi/brokers/account_managers/sql_account_manager.go

```diff
@@ -198,8 +198,8 @@ func (b *SqlAccountManager) BuildInstanceCredentials(bindRecord models.ServiceBi
 		combinedCreds["uri"] = fmt.Sprintf("mysql://%s:%s@%s/%s?ssl_mode=required",
 			url.QueryEscape(combinedCreds["Username"]), url.QueryEscape(combinedCreds["Password"]), combinedCreds["host"], combinedCreds["database_name"])
 	} else if service_to_name[sid] == models.CloudsqlPostgresName {
-		combinedCreds["uri"] = fmt.Sprintf("postgres://%s/%s?user=%s&password=%s&ssl=true",
-			combinedCreds["host"], combinedCreds["database_name"], url.QueryEscape(combinedCreds["Username"]), url.QueryEscape(combinedCreds["Password"]))
+		combinedCreds["uri"] = fmt.Sprintf("jdbc:postgres://%s:%s@%s/%s?sslmode=require&sslcert=%s&sslkey=%s&sslrootcert=%s",
```

Comment:
This discussion has probably happened elsewhere- but:

This is a JDBC specific format and we advertise this a a generic URI. Should be exposed differently in the combinedCreds (binding credentials)? For example as 'jdbc_uri'? 

If it needs to be called URI perhaps URI Format needs to be a binding parameter so the user would so something like:
```bash
cf bind-service myapp mydb -c '{"uri_format":"jdbc"}'
```

Reply:
That's a good idea. It does to need to be called "uri", because if there is a field called "uri", spring cloud connectors will use that. If there isn't a field called that, it'll looks for a field called "jdbcUrl". Maybe the best solution is to include a uri_format parameter to bind, and then exclude the "uri" field if that option is set, and instead pass back "jdbcUrl". What do you think?

Reply:
Sounds good to me!

Reply:
Thanks Jeff!